.github/workflows/deploy.yml

```diff
@@ -77,6 +77,30 @@ jobs:
         with:
           ssh-private-key: ${{ secrets.DEPLOY_SSH_PRIVATE_KEY }}
 
+      - name: Push environment secrets
+        run: bin/kamal env push
+        env:
+          RAILS_SERVE_STATIC_FILES: true
+          RUBY_YJIT_ENABLE: 1
+          DEPLOY_LETSENCRYPT_EMAIL: ${{ secrets.DEPLOY_LETSENCRYPT_EMAIL }}
+          DEPLOY_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          DEPLOY_REGISTRY: ${{ env.REGISTRY }}
+          DEPLOY_SSH_HOST: ${{ secrets.DEPLOY_SSH_HOST }}
+          DEPLOY_USERNAME: ${{ github.actor }}
+          DEPLOY_WEB_HOST: ${{ secrets.DEPLOY_WEB_HOST }}
```

Comment:
I'm not 100% sure that these `DEPLOY_` ones need to be pushed 🤔

Reply:
removed - thanks